Update sandman_de.ts

bastik-1001 opened the pull request:
Fresh fork to address new and old strings.

isaak654 commented on SandboxiePlus/SandMan/sandman_de.ts:
```diff
@@ -3479,22 +3479,22 @@ Notiz: Die Updateprüfung ist oft zeitversetzt zu den letzten GitHub-Veröffentl
         <location filename="Windows/SettingsWindow.cpp" line="1095"/>
         <source>%1 (Current)</source>
-        <translation type="unfinished"></translation>
+        <translation type>%1 (Aktuell)</translation>
```
```suggestion
        <translation>%1 (Aktuell)</translation>
```

bastik-1001 commented:
My intention was to resolve the issue, by clicking on 'Resolve', but that did not what I expected, instead it got marked as resolved, so I fixed my file. There was a spelling mistake I fixed on top. Luckily, all checks have passed now.

I definitely made an error, and now that I saw it, I expected the build process to throw an error about a misformated file, not that the whole build process was going to fail.
